### ISSUE TITLE
New designer search/parameters panel width conflicts with other components

### ISSUE THREAD

**Issue by chmancha** [Azure/LogicAppsUX#5916]
### Describe the issue

1. Go to https://portal.microsofticm.com/imp/v5/automation/workflows
2. Click on "Create new workflow"
3. Add a trigger and see how the panel opens (panel to search trigger/action)
4. Reduce window dimensions and see how the panel conflicts with the left IcM nav bar (screenshots below)
5. This also occurs when clicking on an action and seeing the parameters/settings panel (which is a different setting with the same width value)

I expect the width of the panel to respect the parent width. I have a suggestion on what to change to fix it (described below).

### Are you using new designer or old designer

New Designer (With action panels)

### Screenshots or Videos

This is the visual issue:
The width of the panel overlaps with the IcM navigation bar on the left side. Red line should mark the left end of the width of the panel.
 ![Image](https://github.com/user-attachments/assets/41f738c5-98f2-41cb-b1b9-20c8ada6adf6)

I can see that there's a hard set value for the width of the panel, being set to '630px'. This is set on the Logic Apps code (not coming from any fluent library).
![Image](https://github.com/user-attachments/assets/43c3edd2-b68d-46ba-ab6b-4ff3c9f12d97)

The fix its quite simple, just update the width value to be '100%' and add a max-width: '630px'. That way the behavior remains the same for everyone that doesn't have the conflict, but it fixes the issue for everyone who has a width conflict.

IMPORTANT: This issue occurs in both panels the search trigger/action one as well as the parameters one once you click on the settings of an action. I have to point it out since I noticed the width value is set independently in both panels. Meaning the '630px' width is being set in two places most likely, so both would need to be updated.

Action parameters panel

![Image](https://github.com/user-attachments/assets/837f721a-5ac7-4e5f-823a-5495466a5a52)

![Image](https://github.com/user-attachments/assets/26eaccce-b924-4b0f-9030-cae5028b8b83)

Please let me know if there's anything I can help with

Designer version:

"@microsoft/logic-apps-designer": "4.85.2",

"@microsoft/logic-apps-shared": "4.85.2",

### Browser

Edge Dev
